### PR TITLE
Feat/multiple track audios

### PR DESCRIPTION
**Pull request by luiscovelo**

### CODE REVIEW

File: internal/mosaic/command/builder.go [hunk 1]
```diff
@@ -41,11 +41,19 @@ func Build(m mosaic.Mosaic, cfg *config.Config) []string {
...
-	for _, m := range m.Medias {
+	audioInputs := []string{}
+	for i, media := range m.Medias {
 		videoInputs = append(
 			videoInputs,
-			"-i", m.URL,
+			"-i", media.URL,
 		)
+
+		if m.WithAudio {
```
Comment (mauricioabreu) [inline internal/mosaic/command/builder.go]: Most of the mosaics I have ever seen in my entire life have only the audio of the first video.
Maybe we need this option to have multiple choices:
* No audio (`no_audio`)
* First input only (`first_input`)
* All inputs (`all_inputs`)

**Review by mauricioabreu — CHANGES_REQUESTED** [review]
Good job so far.

File: internal/mosaic/command/builder.go (hunk 1 above)
Comment (luiscovelo) [reply internal/mosaic/command/builder.go]: Oh, right!

Theses options can be a new struct, like as "Audio"?

Example:

```go
type Audio struct {
	NoAudio    bool `json:"no_audio"`
	FirstInput bool `json:"first_input"`
	AllInputs  bool `json:"all_inputs"`
}
```
or 

```go
type Audio struct {
	Off        bool `json:"off"`
	FirstInput bool `json:"first_input"`
	AllInputs  bool `json:"all_inputs"`
}
```

What's your option about this?

Comment (mauricioabreu) [reply internal/mosaic/command/builder.go]: The field could be an enum with these 3 options.
Imagine you are creating a mosaic through an admin. This admin could provide a dropdown with these 3 options

Comment (luiscovelo) [reply internal/mosaic/command/builder.go]: Right,

Like this:

```go
type Audio string

const (
	NoAudio    string = "no_audio"
	FirstInput string = "first_input"
	AllInputs  string = "all_inputs"
)

type (
	Position struct {
		X int
		Y int
	}
	Media struct {
		URL      string `json:"url"`
		Position Position
		Scale    string
	}

	Mosaic struct {
		Name          string  `json:"name"`
		BackgroundURL string  `json:"background_url"`
		Medias        []Media `json:"medias"`
		Audio         Audio   `json:"audio"`
	}
)
```

Using the `Audio` as a type, we can to assign three methods to validate audio options, such as: IsNoAudio(), IsFirstInput() or IsAllInputs().

```go
type Audio string

func (a Audio) IsNoAudio() bool {
	return a == Audio(NoAudio)
}

func (a Audio) IsFirstInput() bool {
	return a == Audio(FirstInput)
}

func (a Audio) IsAllInputs() bool {
	return a == Audio(AllInputs)
}
```

What do you think?

Comment (mauricioabreu) [reply internal/mosaic/command/builder.go]: I agree with the constant approach and the Audio type, pretty cool!

Comment (luiscovelo) [reply internal/mosaic/command/builder.go]: Good, i'll fixed it.

**Comment by luiscovelo** [comment]
@mauricioabreu,

Based in our conversations, i fixed to use a Audio as a enum type.

**Comment by mauricioabreu** [comment]
@luiscovelo good! How did it look like in the player? Did it play smooth with the multiple audios option?

**Comment by luiscovelo** [comment]
@mauricioabreu,

I tested using VLC player, I had to download a mosaic content from bucket and to import on VLC.

I'll to add this options in the HLS player into another PR, I need to evaluate in how to make in HLS.js library, but in VLC player worked very well.

**Review by mauricioabreu — APPROVED** [review]
Good job!

**Pull request merged**